src/ui/RadioTileGroup/RadioTileGroup.tsx

```diff
@@ -0,0 +1,102 @@
+import * as RadioGroupPrimitive from '@radix-ui/react-radio-group'
+import { cva, VariantProps } from 'cva'
+import React from 'react'
+
+const group = cva(['flex', 'gap-4'], {
+  variants: {
+    direction: {
+      row: 'flex-row',
+      col: 'flex-col',
+    },
+  },
+  defaultVariants: {
+    direction: 'row',
+  },
+})
+interface GroupProps
+  extends React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Root>,
+    VariantProps<typeof group> {}
+
+const Group = React.forwardRef<
+  React.ElementRef<typeof RadioGroupPrimitive.Root>,
+  GroupProps
+>(({ className, direction, ...props }, ref) => {
+  return (
+    <RadioGroupPrimitive.Root
+      className={group({ className, direction })}
+      {...props}
+      ref={ref}
+    />
+  )
+})
+Group.displayName = RadioGroupPrimitive.Root.displayName
+
+const item = cva(['relative'], {
+  variants: {
+    flex: {
+      1: 'flex-1',
+      none: 'flex-none',
+    },
+  },
+  defaultVariants: {
+    flex: 1,
+  },
+})
+interface ItemProps
+  extends React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Item>,
+    VariantProps<typeof item> {
+  label: string
+  description?: string
+}
+
+const Item = React.forwardRef<
+  React.ElementRef<typeof RadioGroupPrimitive.Item>,
+  ItemProps
+>(({ className, label, description, flex, ...props }, ref) => {
```

Comment:
Considered using children as the label. Lmk if you think that's better. Chose not to bc seemed weird to have label as children but not description. Can't really have both with current markup but could probably figure out how to get it to work.

Reply:
I can see a way to get this to work ... however we will have to ensure that when developing people put them in the correct order.

You'd have your label component as:

```tsx
<div className="flex items-center justify-between gap-4">
  <p className="font-medium">{children}</p>
  <RadioButtonCircle />
</div>
```

Description component as:

```tsx
{description ? (
  <p className="text-left text-ds-gray-quinary">{children}</p>
) : null}
```

And than the radio group as:

```tsx
<div className="flex h-full flex-col gap-2 rounded-md border border-ds-gray-quaternary p-4">
  {children}
</div>
```

Reply:
This way to we get a bit finer controls over the classnames for the label and description if there's a one off request